app/javascript/optimization/optimization_list.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { DataDrivenTable } from '../components/simple-table/simple-table.jsx';
+import { miqOptimizationTransform } from '../optimization/transform.js';
+import { refresh } from './refresh.js';
+
+const empty = () => (
```

Comment:
I would make a Component from this. `Empty` => `<Empty />`

Reply:
fixed